dist/templates/ovnkube-alerts.yaml.j2

```diff
@@ -191,6 +191,17 @@ spec:
          The 99th percentile {{ $labels.name }} update latency {{ value }} aggregated
          across all masters for the last 15minutes is more than 1 seconds.
 
+    - alert: OvnKubeNodeLogFileSize
+      expr: |
+        rate(ovnkube_node_logfile_size{namespace="ovn-kubernetes", job="ovnkube-node"}[15m]) * 5 * 60 / 1024 / 1024 > 1
```

Comment:
@girishmg can you please put a comment in the yaml explaining what your math is calculating?

Reply:
Will do.

Reply:
Breaking this down:
`rate(ovnkube_node_logfile_size{namespace="ovn-kubernetes", job="ovnkube-node"}[15m])` - per second change in bytes over last 15 mins

`* 5 * 60` convert to per 5 min 

`/ 1024 / 1024` convert to MB

`> 1` greater than 1 MB

`for: 10m` only fire if the expression above is consistently true for 10m 

I have no understanding for how much bytes we write to log files in times of great load.

Reply:
yup, that is essentially what we are doing. Added a comment to that effect.